Unity/Assets/GameSystems/Unity/Components/Movement/Jump2DRB.cs

```diff
@@ -0,0 +1,34 @@
+using UnityEngine;
+using UnityLayer;
+using Zenject;
+
+[RequireComponent(typeof(Rigidbody2D))]
+public class Jump2DRB : MonoBehaviour, IJump2D
+{
+    [SerializeField] float force;
```

Comment:
Need a private after the [serializeField], as mentioned here - https://github.com/AdolfoPSakata/UnityTraining-2/pull/1